### PR TITLE
Feature/bazooka/chat

### PR DESCRIPTION
**Pull request by sladan-hedvig**
- Reworked chat

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/App/Sources/Journeys/LoggedInJourney.swift [hunk 1]
```diff
@@ -65,6 +67,15 @@ extension AppJourney {
             .configureSubmitClaimsNavigation
             .configurePaymentNavigation
             .configureContractNavigation
+            .configureChatNavigation
+        //            .onPresent {
+        //                DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```
Comment (julia-andersson) [inline Projects/App/Sources/Journeys/LoggedInJourney.swift:72]: remove comment

File: Projects/Chat/Sources/Views/ImagesView.swift [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+                                    )
+                                    inCont.resume(returning: file)
+                                })
+                        }
+                case .image:
+                    guard let uti = contentInput?.uniformTypeIdentifier else {
+                        inCont.resume(throwing: GenerateFileUploadError.failedToGenerateMimeType)
+                        return
+                    }
+                    //
```
Comment (julia-andersson) [inline Projects/Chat/Sources/Views/ImagesView.swift:228]: remove

File: Projects/Chat/Sources/Views/ChatScreenViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+                await self?.fetch()
+            }
+        }
+        Task { [weak self] in
+            await self?.fetch()
+        }
+        let fileUploadManager = FileUploadManager()
+        fileUploadManager.resetuploadFilesPath()
+        DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
+            AskForRating().ask()
```
Comment (julia-andersson) [inline Projects/Chat/Sources/Views/ChatScreenViewModel.swift:42]: Is this one ever triggered? Does not seem like it. Maybe we can either we can remove this or make it work?

**Review by julia-andersson — CHANGES_REQUESTED** [review]

File: Projects/Chat/Sources/Views/ChatScreenViewModel.swift (hunk 3 above)
Comment (sladan-hedvig) [reply Projects/Chat/Sources/Views/ChatScreenViewModel.swift:42]: It is working, on third time if you check implementation
```
 func ask() {
        guard !UserDefaults.standard.bool(forKey: userDefaultsCompletedKey) else { return }

        let numberOfSessions = UserDefaults.standard.value(forKey: "AskForRating") as? Int ?? 0

        if numberOfSessions >= 3 {
            UserDefaults.standard.set(true, forKey: userDefaultsCompletedKey)
            if let scene = UIApplication.shared.currentScene {
                SKStoreReviewController.requestReview(in: scene)
            }
        }
    }
```

**Review by julia-andersson — APPROVED** [review]

**Pull request merged**